en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md

```diff
@@ -0,0 +1,237 @@
+# Email Connector Example
+
+Email Connector can be used to perform operations using standard protocols such as SMTP, IMAP and POP3. 
+
+## What you'll build
+
+This example explains how to use Email Connector to send an email and retrieve the same email from GMail. The user sends a payload with the recipients, content and attachments of the email. Then, by invoking another API resource, the content of the sent email will be retrieved. 
+
+It will have two HTTP API resources, which are `send` and `retrieve`. 
+
+* `/send `: The user sends the request payload which includes the recipients, content and attachments of the email. This request is sent to WSO2 EI by invoking the EmailConnector API. It will send the email to the relevant recipients. 
```

Comment:
`/send `: This API resource will retrieve information from the incoming HTTP post request such as recipients, content and attachments and construct the email and send to the mentioned recipients.

Reply:
Added to the section where the API resource is being added